### PR TITLE
feat: Reduced analytics v2

### PR DESCRIPTION
**Pull request by schinkowski**
This should do what https://github.com/unity3d-jp/MeshSync/pull/807/files does but adapted to work with the refactored code.

### CODE REVIEW

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs [hunk 1]
```diff
@@ -49,6 +50,26 @@ internal abstract class AnalyticsDataHandlerBase {
         protected virtual int version { get; }
         protected virtual int eventsPerHour => 10000;
 
+        protected abstract Dictionary<string, DateTime> lastSendTimes { get; }
+
+        protected virtual TimeSpan MinTimeBetweenEvents {
+            get { return TimeSpan.FromSeconds(10); }
```
Comment (SonnyCampbellUnity) [inline Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:56]: This should be 1 second, not 10 seconds right?

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs [hunk 2]
```diff
@@ -146,14 +177,17 @@ struct SyncEventData {
                 syncMode = syncData.syncMode
             };
 
-            Send(eventData);
+            Send(eventData, entityTypeStr);
```
Comment (seandillon92) [inline Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs]: Can you have both Asset and Entity types in data? what happens if entityType is none and Asset type has a value, should we not handle it and send the asset type instead?

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs (hunk 1 above)
Comment (schinkowski) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:56]: It's 10 seconds in Colm's PR. Can set it to anything we want though. How much do we need to reduce the analytics by?

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs (hunk 2 above)
Comment (SonnyCampbellUnity) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs]: Yeah correct, we need to handle both asset type and entity type.

If you look at Colm's PR you should be able see what needs to be handled

Comment (schinkowski) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs]: Yeah probably, fixed now

**Review by seandillon92 — DISMISSED** [review]

File: Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs (hunk 1 above)
Comment (SonnyCampbellUnity) [reply Editor/Scripts/Analytics/MeshsyncAnalyticsHandlers.cs:56]: I think we are supposed to send an event every 1 second

**Review by seandillon92 — DISMISSED** [review]

**Pull request closed without merging**